tests/scripts/run_tests.sh

```diff
@@ -379,7 +379,7 @@ set_up_chdir() {
     # The user might have multiple entries in their PYTHONPATH so we should try to find the right one
     IFS=':' read -ra ENTRIES <<< "$PYTHONPATH"
     for ENTRY in "${ENTRIES[@]}"; do
-      if [[ $ENTRY == *"tt-metal" ]]; then
+      if [[ $ENTRY == *"third-metal" ]]; then
```

Comment:
Is this change intended? If so why is it needed?

Reply:
Do you work on two side `metal`s???